pkg/types/file_error.go

```diff
@@ -6,7 +6,7 @@ import (
 
 // ParsedFile contains the parsed content and any error encountered during processing of a file.
 //
-//nolint:errname
+//nolint:errname // Error as part of struct for simplification
```

Comment:
But could be an explicit field right? Let's change that then.

Reply:
It is an explicit filed already, but the linter does not like that we name it `err`. Do you have a better name suggestion